float32 rad
float32 azi
float32 ele
float32 temperature
float32 max_temperature
int32 points
int32 index
